Makefile

```diff
@@ -95,7 +95,7 @@ release-announce: .release-functest
 cluster-up:
 	./cluster-up/up.sh
 
-cluster-down:
+cluster-down: cluster-clean
```

Comment:
I fear that `cluster-clean` is not something we would want to depend on `cluster-down`. `cluster-down` is supposed to forcefully destroy all traces of the cluster. `cluster-clean` is a graceful cleanup which is used between redeploys of an existing cluster. It may stall or exit with an error, which is completely fine in the redeploy case.

Reply:
moreover as we discussed, in case new code is pushed, we cant count on the 5 minutes graceful shutdown (out of this PR scope)

Reply:
How do you solve then the test finishing in a graceful way? On kind, it leaks and may leak in the future other things. We need a way to clean after the tests are done.

I'm fine if we do not handle the cleanup target in this PR and follow it later on.
I do want the other stuff in.

Reply:
You could instead try to explicitly delete this pod in `cluster-up/down.sh` when running on kind. Let's ensure that non-kind providers don't try that. This is a workaround just for kind and may cause issues for other providers otherwise.

Reply:
@zhlhahaha can you please take this line out so we can take the change in?

Reply:
> You could instead try to explicitly delete this pod in `cluster-up/down.sh` when running on kind. Let's ensure that non-kind providers don't try that. This is a workaround just for kind and may cause issues for other providers otherwise.

OK, thanks.
We can take care of it in a following PR if needed (unless @zhlhahaha wants t invest further on this now).

Reply:
> > You could instead try to explicitly delete this pod in `cluster-up/down.sh` when running on kind. Let's ensure that non-kind providers don't try that. This is a workaround just for kind and may cause issues for other providers otherwise.
> 
> OK, thanks.
> We can take care of it in a following PR if needed (unless @zhlhahaha wants to invest further on this now).

Of course, I can give it a try. Thanks for your patient.

Reply:
I will create another PR for the following up.